### PR TITLE
feat(webpack): experimental flag for next 13 app directory

### PR DESCRIPTION
**Pull request by idoros**
This PR adds an experimental flag `experimentalAttachCssToContainingChunks` that changes the way the CSS asset is attached to webpack chunks - in the default (off mode), the result generated CSS asset is attached to entrypoint chunks, and when the flag is enabled, it attaches the CSS to every chunk that contains stylable stylesheet.  

This is primarily added to test a fix for Next 13 new experimental [app directory feature](https://beta.nextjs.org/docs/routing/fundamentals#the-app-directory) (raised in #2773). In the default mode Next resolve the attached generated CSS module mistakenly as a JavaScript file and attach it in the HTML as a script tag. 

**Notice 1:** I'm adding this experimental flag that seems to resolve the issue, although unfortunately nor NextJS or Webpack document how chunks are used in a clear way or not at all. So this will probably need more investigations and fixes in the future.    

**Notice 2:** There are no tests for this in this PR as there is no NextJS integration in the Stylable repo - we might want to add a separate repository to test such integrations without adding more dependencies.

also added Stylable.io documentation about this configuration: https://github.com/wixplosives/stylable.io/pull/64

### CODE REVIEW

**Review by barak007 — APPROVED** [review]

**Comment by barak007** [comment]
We want to write docs for that in next integration page

**Comment by idoros** [comment]
Added a PR to stylable.io: https://github.com/wixplosives/stylable.io/pull/64

**Review by barak007 — APPROVED** [review]

**Review by barak007 — APPROVED** [review]

File: packages/webpack-plugin/src/plugin.ts [hunk 1]
```diff
@@ -142,6 +142,11 @@ export interface StylableWebpackPluginOptions {
      * Defaults to true.
      */
     includeGlobalSideEffects?: boolean;
+    /**
+     * Experimental flag that attaches CSS bundle asset to every chunk that contains references to stylable stylesheets.
+     * The default off mode attaches the only to entry chunks.
```
Comment (tomrav) [inline packages/webpack-plugin/src/plugin.ts:147]:
```suggestion
     * The default off mode attaches only to entry chunks.
```

**Review by tomrav — APPROVED** [review]
Looks good!

**Pull request merged**